lib/web/css/source/lib/_buttons.less

```diff
@@ -73,7 +73,7 @@
     .lib-css(padding, @_button-padding);
     .lib-css(width, @_button-width);
     .lib-font-size(@_button-font-size);
-    .lib-line-height(@_button-line-height);
+    .lib-css(line-height, @_button-line-height);
```

Comment:
This is just working around #25592 – if there are downstream themes that use `.lib-line-height(@_button-line-height)` directly (probable) they'll be very surprised when things look bad on updating.

I personally don't think that `@button__line-height__base` should be updated to unitless until that issue is resolved.

Reply:
i have reverted back this file . We should do this in another seperate PR